conda/cli/main.py

```diff
@@ -37,12 +37,29 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 import sys
 
+PARSER = None
+
 
 def generate_parser():
+    global PARSER
```

Comment:
Got it. I sent commented and then immediately thought "oh, right, he might have done it because of imports!" :D -- so, never mind, thanks to confirm!